cloud-storage/hex.c

```diff
@@ -0,0 +1,59 @@
+/* Copyright  (C) 2019 The RetroArch team
+ *
+ * ---------------------------------------------------------------------------------------
+ * The following license statement only applies to this file (hex.c).
+ * ---------------------------------------------------------------------------------------
+ *
+ * Permission is hereby granted, free of charge,
+ * to any person obtaining a copy of this software and associated documentation files (the "Software"),
+ * to deal in the Software without restriction, including without limitation the rights to
+ * use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software,
+ * and to permit persons to whom the Software is furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+ * INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+ * IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY,
+ * WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+#include <stdlib.h>
+
+#include "hex.h"
+
+static char hex_table[] = {
+   '0',
+   '1',
+   '2',
+   '3',
+   '4',
+   '5',
+   '6',
+   '7',
+   '8',
+   '9',
+   'a',
+   'b',
+   'c',
+   'd',
+   'e',
+   'f'
+};
+
+char *bytes_to_hex_str(unsigned char *bytes, uint64_t len)
```

Comment:
Wanna move this to libretro-common/string/stdstring.c (if it's not there already)? I think this might already exist, but I'm quite new to RA, so I'm just assuming

Reply:
I'll take a look and remove this function from here.

Reply:
Moved to stdstring.